### ISSUE TITLE
Expanded View in Home Assistant?

### ISSUE THREAD

**Issue by mboarman** [ljmerza/ha-our-groceries#11]
Can this be setup with a flag to always show in expanded view in home assistant? I would like to be able to keep in on a wall mounted panel in the kitchen and quickly see if an item is on the list without having to expand the card each time? 

If not any suggestions on where/how I might edit the script to accomplish this?

Thanks in advance! I

**Comment by AsWaM** [comment]
Hello @mboarman 
Do you manage to acces the content of the shopping lists? If yes how, with which card?

**Comment by mboarman** [comment]
@AsWaM Yes, I use the integration along with the lovelace card.  I would just like an option for the card to show the expanded list instead of having to click on the list each time to view the contents. Thanks for the consideration!!

**Comment by ljmerza** [comment]
the issue with this is, you don't get the details of a list unless you ask for it. So for this to work you'd have to request the IDs of the lists, then request the list details for each list...everytime. Considering the python module is an unofficial API, I'm hesitant to abuse the 'API' from ourgroceries and they come down on the python library.

**Comment by d-edge82** [comment]
> @AsWaM Yes, I use the integration along with the lovelace card. I would just like an option for the card to show the expanded list instead of having to click on the list each time to view the contents. Thanks for the consideration!!

which card is it please?
